This patch removes dependencies on other Puppet modules which do not
apply to Solaris.  This patch is not suitable for upstream
contribution.

--- puppetlabs-rsync-0.4.0/metadata.json.orig	2015-05-13 14:49:30.562109445 -0600
+++ puppetlabs-rsync-0.4.0/metadata.json	2015-05-13 14:49:41.689334442 -0600
@@ -9,7 +9,6 @@
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
   "dependencies": [
     {"name":"puppetlabs/stdlib","version_requirement":">=2.2.1"},
-    {"name":"puppetlabs/xinetd","version_requirement":">=1.1.0"},
     {"name":"puppetlabs/concat","version_requirement":">= 1.1.1"}
   ]
 }
